x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps.tsx

```diff
@@ -11,13 +11,15 @@ import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
 
 import type { AgentPolicy, PackagePolicy } from '../../types';
-import { sendGetOneAgentPolicy } from '../../hooks';
+import { sendGetOneAgentPolicy, useKibanaVersion } from '../../hooks';
 import { FLEET_SERVER_PACKAGE } from '../../constants';
 
 import { EnrollmentStepAgentPolicy } from './agent_policy_selection';
 import { AdvancedAgentAuthenticationSettings } from './advanced_agent_authentication_settings';
 
 export const DownloadStep = () => {
+  const kibanaVersion = useKibanaVersion();
+  const kibanaVersionURLString = useMemo(() => kibanaVersion.replace(/\./g, '-'), [kibanaVersion]);
```

Comment:
IIRC the Kibana version can include a suffix like `7.14.0-SNAPSHOT`, it will probably be safer to use `semver` to parse & retrieve the parts